plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/BaseElasticsearchConnectorTest.java

```diff
@@ -499,6 +499,103 @@ public void testMixedArray()
                 "VALUES NULL, ARRAY['hello'], ARRAY['foo', 'bar']");
     }
 
+    @Test
+    public void testMetaPrestoNamespaceBackwardsCompatibility()
+            throws IOException
+    {
+        String indexNamePresto = "test_meta_backwards_compatibility_presto";
+
+        @Language("JSON")
+        String mappingPresto = "" +
+                "{" +
+                "      \"_meta\": {" +
+                "        \"presto\": {" +
+                "          \"a\": {" +
+                "                \"isArray\": true" +
+                "          }" +
+                "        }" +
+                "      }," +
+                "      \"properties\": {" +
+                "        \"a\": {" +
+                "          \"type\": \"keyword\"" +
+                "        }" +
+                "      }" +
+                "}";
+
+        createIndex(indexNamePresto, mappingPresto);
+
+        index(indexNamePresto, ImmutableMap.<String, Object>builder()
+                .put("a", ImmutableList.of("foo", "bar"))
+                .build());
+
+        assertQuery(
+                "SELECT a FROM test_meta_backwards_compatibility_presto",
+                "VALUES ARRAY['foo', 'bar']");
```

Comment:
This code snippet is repeated (with slight modifications) 3 times within this test.
Consider using 

`@Test(dataProvider = ...`

(see for reference `io.trino.testing.AbstractTestDistributedQueries#testCaseSensitiveDataMapping`)
to avoid the code duplications.

Reply:
unrelated to this change though so i'm going to forgo this. We will be refactoring tests for elasticsearch shortly